README.md

```diff
@@ -51,6 +51,59 @@ zwadp -c iso --oid oid_m31.dat --oid fakes/oid_m31_fake.dat --feature feature_m3
 zwadp -h
 ```
 
+### Active anomaly detection
+
+Lets use the same data from [Zenodo](https://zenodo.org/record/4318700).
+```shell
+cd data
+zwad-zenodo
+```
+
+It is better to create temporary directory to run active anomaly detection:
+```shell
+mkdir ../tmp
+cd tmp
+```
+
+```shell
+# Run the algorithm with M31 dataset
+zwaad --oid ../data/oid_m31.dat --feature ../data/feature_m31.dat --feature-names ../data/feature_m31.name --budget 4
+```
+
+The script output may be as the following:
+```
+Check https://ztf.snad.space/dr4/view/695211400124577 for details
+Is 695211400124577 anomaly? [y/N]:
+```
+It is waiting for our decision now. Let us decide that `695211400124577` is an anomaly and continue:
+```
+Check https://ztf.snad.space/dr4/view/695211400124577 for details
+Is 695211400124577 anomaly? [y/N]: y
+Check https://ztf.snad.space/dr4/view/695211400034403 for details
+Is 695211400034403 anomaly? [y/N]: n
+Check https://ztf.snad.space/dr4/view/695211400070144 for details
+Is 695211400070144 anomaly? [y/N]: n
+Check https://ztf.snad.space/dr4/view/695211400134512 for details
+Is 695211400134512 anomaly? [y/N]: y
+UNK  4
+```
+
+Now we have examined all four samples within our budget.
+```shell
+# Check found anomalies
+cat ./data/aad/anomalies_feature_m31.txt
+695211400124577
+695211400134512
+
+# Check answers log
+cat ./data/aad/answers_feature_m31.csv
+oid,is_anomaly
+695211400124577,1
+695211400034403,0
+695211400070144,0
+695211400134512,1
+```
```

Comment:
After I run it I have the only new file `my_anomalies.txt`, no CSV file. And it is located in the current working directory, not in `data`